tests/dialects/test_redshift.py

```diff
@@ -181,6 +181,16 @@ def test_identity(self):
         )
 
     def test_values(self):
+        self.validate_all(
+            "SELECT * FROM (VALUES (1, 2)) AS t",
+            write={"redshift": "SELECT * FROM (SELECT 1, 2) AS t"},
+        )
+        self.validate_all(
+            "SELECT * FROM (VALUES (1)) AS t1(id) CROSS JOIN (VALUES (1)) AS t2(id)",
```

Comment:
Probably a rare use case of `VALUES`, but thought I'd include it anyway. We weren't removing the second `VALUES` clause because only `exp.From` was checked as a possible parent.